src/Bundle/Maker/Factory/NoPersistanceObjectsAutoCompleter.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Zenstruck\Foundry\Bundle\Maker\Factory;
+
+final class NoPersistanceObjectsAutoCompleter
+{
+    public function __construct(private string $kernelRootDir)
+    {
+    }
+
+    public function getAutocompleteValues(): array
+    {
+        $classes = [];
+
+        $excludedFiles = $this->excludedFiles();
```

Comment:
the file `src/functions.php` was problematic: when transforming this file into its PSR4 representation, if I call `class_exists()`, I got an error I can't even catch as a `\Throwable` :shrug: 

```
Fatal error: Cannot redeclare Zenstruck\Foundry\factory() (previously declared in /app/src/functions.php:16) in /app/src/functions.php on line 16
```

then I decided to exclude all files found in composer's config under `autoload.files` node